### PR TITLE
Dev/heroku setup

### PR DESCRIPTION
**Pull request by gDev95**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -6,22 +6,31 @@
...
-  },
-  "devDependencies": {
-    "@types/express": "^4.17.0",
-    "@types/node": "^12.7.0",
+    "nodemon-webpack-plugin": "^3.0.1",
+    "ts-loader": "^6.0.4",
     "ts-node": "^8.3.0",
-    "typescript": "^3.5.3"
-  }
+    "ts-node-dev": "^1.0.0-pre.40",
```
Comment (harastaivan) [inline backend/package.json]: Should we distinguish between prod and dev dependencies?

File: frontend/package.json [hunk 2]
```diff
@@ -39,19 +53,5 @@
...
-    "babel-jest": "^24.8.0",
-    "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.14.0",
-    "identity-obj-proxy": "^3.0.0",
-    "react-test-renderer": "^16.8.6",
-    "ts-jest": "^24.0.2",
-    "tslint": "^5.18.0",
-    "typescript": "^3.5.3"
-  }
+  "devDependencies": {}
```
Comment (harastaivan) [inline frontend/package.json]: Why moved?

File: package.json [hunk 3]
```diff
@@ -3,7 +3,23 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "install": "yarn install:backend && yarn install:frontend",
+    "install:backend": "cd backend && yarn install",
```
Comment (harastaivan) [inline package.json]: `yarn install --prefix backend` works too. Depends what we prefer.

File: backend/package.json (hunk 1 above)
Comment (gDev95) [reply backend/package.json]: Ahh i wish but heroku is a little weird about it. Its either we turn of prod mode in heroku or we do it this way

File: frontend/package.json (hunk 2 above)
Comment (gDev95) [reply frontend/package.json]: can be removed for now

**Pull request merged**